shared/teams/team/member/index.new.tsx

```diff
@@ -170,43 +171,97 @@ const SubteamNotInRow = (props: SubteamNotInRowProps) => {
         users: [{assertion: props.username, role}],
       })
     )
+  const openTeam = React.useCallback(
+    () =>
+      dispatch(
+        nav.safeNavigateAppendPayload({
+          path: [{props: {teamID: props.subteam.id}, selected: 'team'}],
+        })
+      ),
+    [props.subteam.id, dispatch, nav]
+  )
 
   const disabledRoles = Container.useSelector(state =>
     Constants.getDisabledReasonsForRolePicker(state, props.subteam.id, props.username)
   )
 
   const [role, setRole] = React.useState<Types.TeamRoleType>('writer')
   const [open, setOpen] = React.useState(false)
-  const action = (
-    <FloatingRolePicker
-      onConfirm={() => {
-        onAdd(role)
-        setOpen(false)
-      }}
-      onSelectRole={setRole}
-      open={open}
-      onCancel={() => setOpen(false)}
-      selectedRole={role}
-      disabledRoles={disabledRoles}
-    >
-      <Kb.Button label="Add" onClick={() => setOpen(!open)} />
-    </FloatingRolePicker>
-  )
 
   const memberCount = props.subteam.memberCount ?? -1
-  const body = (
-    <Kb.Box2 direction="vertical" alignItems="flex-start">
-      <Kb.Text type="BodySemibold">{props.subteam.teamname}</Kb.Text>
-      <Kb.Text type="BodySmall">
-        {memberCount.toLocaleString()} {pluralize('member', memberCount)}
-      </Kb.Text>
+
+  return (
+    <Kb.Box2 direction="vertical" fullWidth={true} style={styles.rowCollapsedFixedHeight}>
```

Comment:
maybe add a comment here to replace this with a ListIem2 when we fix the padding issues on it

Reply:
I honestly don't know if we want to do it at this point. The expansion stuff seems impossible to properly implement in a LI2 :/